service.toml

```diff
@@ -21,4 +22,7 @@ optional = ["content_md5", "content_type", "io_callback"]
 
 [pairs.default_storage_pairs]
 type = "DefaultStoragePairs"
-description = "set default pairs for storager actions"
\ No newline at end of file
+description = "set default pairs for storager actions"
+
+[infos.object.meta.upload-session-id]
+type = "string"
```

Comment:
Please add a link for this metadata: https://www.dropbox.com/developers/documentation/http/documentation#files-upload_session-start